app/pb_kits/playbook/packs/site_styles/docs/_kit_doc.scss

```diff
@@ -143,11 +143,18 @@
   padding: 10px 30px 10px 30px;
   border: 1px solid $border-light;
 
+  &.dark {
+      background: #0a0527;
+      color: white;
```

Comment:
Won't this hide potential problems in our dark kits? I think perhaps adding this only to the markdown sections might be better?